packages/core/database/src/entity-manager/regular-relations.ts

```diff
@@ -220,7 +220,7 @@ const cleanOrderColumns = async ({
     return;
   }
 
-  // Handle databases that don't support window function ROW_NUMBER (here it's MySQL 5)
+  // Handle databases that don't support window function ROW_NUMBER
   if (!strapi.db.dialect.supportsWindowFunctions()) {
```

Comment:
TBH we could remove the entire concept of dialect.supportsWindowFunctions because now all of them do. But I figured I would leave it just in case we support something in the future that doesn't.